concepts/Services/Services.md

```diff
@@ -4,3 +4,4 @@ Services can be thought of as libraries which contain functions that you might w
 
 <docmeta name="displayName" value="Services">
 
+For additional reference <a href="https://blog.sergiocruz.me/sailsjs-services-how-to-use-them-in-your-controllers/">https://blog.sergiocruz.me/sailsjs-services-how-to-use-them-in-your-controllers/</a>
```

Comment:
This is a good reference! Could you change the copy to read as a full sentence, though? Maybe “For additional reference, check out this blog post:”